subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/java/JavaFirstUsePerformanceTest.groovy

```diff
@@ -59,10 +60,10 @@ class JavaFirstUsePerformanceTest extends AbstractCrossVersionPerformanceTest {
         result.assertCurrentVersionHasNotRegressed()
 
         where:
-        testProject                              | _
-        LARGE_MONOLITHIC_JAVA_PROJECT            | _
-        LARGE_JAVA_MULTI_PROJECT                 | _
-        LARGE_JAVA_MULTI_PROJECT_KOTLIN_DSL      | _
+        testProject                         | runs
+        LARGE_MONOLITHIC_JAVA_PROJECT       | null
```

Comment:
We can drop this to 10 as well.

Reply:
Can you explain this a little bit? Is this just rule of thumb?

Reply:
Look at how long they take to run. It's about 25s. With 40 iterations that would be near 20min. We don't need that much time to make a slower/faster decision.

Reply:
So I guess we can also decrease `LARGE_JAVA_MULTI_PROJECT` as well? Now it takes 32min in each run.

Reply:
Yes, absolutely.